frontend/src/routes/project/[project_code]/+page.ts

```diff
@@ -24,10 +28,22 @@ export async function load(event: PageLoadEvent) {
                 }
             }
         }
-`), {projectCode: event.params.project_code}).toPromise();
+`), { projectCode: event.params.project_code }).toPromise();
     if (result.error) throw new Error(result.error.message);
     return {
         project: result.data?.projects[0],
         code: event.params.project_code
     };
+}
+
+export async function _deleteProjectUser(projectId: string, userId: string) {
```

Comment:
is the `_` here meant to convey something private in nature?

Reply:
probably trying to force you to keep something like this in an external module... this file has an established API and it sees what you're doing here as a code smell.  You might think of it as `deleteProjectUser` belonging in an angular service.